Source/plugins/Channel.h

```diff
@@ -153,13 +155,46 @@ namespace PluginHost {
             {
                 return (_current.IsValid() == false);
             }
+
+            inline const string getIdentifier(const char* stream, const uint16_t length)
+            {
+                uint16_t current = 0;
+                bool bValid = false;
+                if ((length > 0) && (stream[current] != '{') && (stream[current] != '[')) {
+                    // look for the delimter, colon (:)
+                    while (current < length) {
+                        if (stream[current] == ':') {
+                            bValid = true;
+                            break;
+                        }
+                        current++;
+                    }
+                }
+
+                auto result = (bValid != false)
```

Comment:
Hi Vinod, I think we did discuss but perhaps if I'm mistaken: I guess it would be better do have this parsed with the core JSON parser (we have the parsing code in one place and it can for example also parse when characters are coming in in chunks, or even one by one)

Reply:
The input is not a valid JSON formatted string. So cannot use JSON parser here. Had a discussion earlier with Pierre on this. There is a plan to re-design this functionality.  This PR is no longer valid and we can close this.